woocommerce/woocommerce-ios
# Hub Menu: design and added options to the tab menu

Part of #5509

### Description
As part of #5509 I implemented the new Hub Menu. In particular:
- Added new icons.
- Implemented a new `WooStyleModifier` for text links.
- Designed the HubMenu view using `LazyVGrid` in SwiftUI.
- Implemented some actions like the `Show View Store` action and `Show reviews` action.
- Implemented every single `HubMenuElement`.
- Implemented the `ReviewsView` conforming to `UIViewControllerRepresentable` for wrapping the `ReviewsViewController`.

### Testing instructions
1. Launch the app in debug mode (so the feature flag for the hub menu will be enabled).
2. Tap the hub menu tab.
3. You should be able to see the new design of the view.
4. The only two actions enabled at this time should be `View Store` and `Reviews`.


### Screenshots
<img src="https://user-images.githubusercontent.com/495617/144622429-77ca2541-2746-44c7-9345-67308253e09a.png" width=300 />

---
